pkg/host/internal/sriov/sriov.go

```diff
@@ -417,25 +430,53 @@ func (s *sriov) ConfigSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *
 
 			if !sriovnetworkv1.StringInArray(group.DeviceType, vars.DpdkDrivers) {
 				if err := s.kernelHelper.BindDefaultDriver(addr); err != nil {
-					log.Log.Error(err, "configSriovDevice(): fail to bind default driver for device", "device", addr)
+					log.Log.Error(err, "configSriovVFDevices(): fail to bind default driver for device", "device", addr)
 					return err
 				}
 				// only set MTU for VF with default driver
 				if group.Mtu > 0 {
 					if err := s.networkHelper.SetNetdevMTU(addr, group.Mtu); err != nil {
-						log.Log.Error(err, "configSriovDevice(): fail to set mtu for VF", "address", addr)
+						log.Log.Error(err, "configSriovVFDevices(): fail to set mtu for VF", "address", addr)
 						return err
 					}
 				}
 			} else {
 				if err := s.kernelHelper.BindDpdkDriver(addr, group.DeviceType); err != nil {
-					log.Log.Error(err, "configSriovDevice(): fail to bind driver for device",
+					log.Log.Error(err, "configSriovVFDevices(): fail to bind driver for device",
 						"driver", group.DeviceType, "device", addr)
 					return err
 				}
 			}
 		}
 	}
+	return nil
+}
+
+func (s *sriov) configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetworkv1.InterfaceExt, skipVFConfiguration bool) error {
```

Comment:
still need to give this one a bit more thought ...

Reply:
We had an offline discussion about it. latest iteration is OK imo.
(had some concerns about how this plays out with Externally managed)